### PR TITLE
Manual Rubocop Fixes

### PR DESCRIPTION
**Pull request by highb**
This should fix the remaining Rubocop issues that couldn't be autocorrected.

Still working on the last couple of issues, I'll remove the DNM tag when those are finished.

### CODE REVIEW

File: lib/vmpooler/pool_manager.rb [hunk 1]
```diff
@@ -369,7 +369,7 @@ def _destroy_vm(vm, pool, provider)
 
         $redis.srem('vmpooler__completed__' + pool, vm)
 
-        finish = format('%.2f', Time.now - start)
+        finish = format('%<time>.2f', time: Time.now - start)
```
Comment (mattkirby) [inline lib/vmpooler/pool_manager.rb]: Presumably this was to resolve a failure. Does the addition of 'time:' change the behavior at all?

Comment (highb) [reply lib/vmpooler/pool_manager.rb]: Yep, the [StyleFormatStringToken](https://rubocop.readthedocs.io/en/latest/cops_style/#styleformatstringtoken) cop prefers named parameters and it shouldn't change the behavior at all. I can verify that the output looks the same now.

Comment (highb) [reply lib/vmpooler/pool_manager.rb]: If you prefer not having named parameters in the format strings, we could also do `EnforcedStyle: unannotated`

Comment (mattkirby) [reply lib/vmpooler/pool_manager.rb]: I'm fine with it, I just wanted to make sure that I followed.

**Pull request merged**